pilot/pkg/features/pilot.go

```diff
@@ -524,6 +524,9 @@ var (
 		true,
 		"If false, TCP probes will not be rewritten and therefor always succeed when a sidecar is used.",
 	).Get()
+
+	EnableInternalTrafficPolicy = env.RegisterBoolVar("PILOT_ENABLE_INTERNAL_TRAFFIC_POLICY", false,
```

Comment:
Kubernetes calls this field `InternalTrafficPolicy` as it's a string that can have "Cluster" or "Local". If you want to make it an boolean this flag should be renamed to something like: `PILOT_ENABLE_NODE_LOCAL_ROUTING`

Reply:
Nope, this flag is like the k8s feature gate, if true, we will consider service spec.InternalTrafficPolicy, otherwise ignore.

Reply:
I see. Thanks for the clarification. Calling it `PILOT_USE_KUBERNETES_INTERNAL_TRAFFIC_POLICY` might be more clear.

Reply:
This conversation is not resolved..is there a reason to not change this env var name?

Reply:
Sorry missing it, now change 'PILOT_ENABLE_INTERNAL_TRAFFIC_POLICY' to 'PILOT_USE_KUBERNETES_INTERNAL_TRAFFIC_POLICY '